settings/eslint/eslintrc.js

```diff
@@ -0,0 +1,96 @@
+const commonParserOptions = {
+  ecmaVersion: '2020',
```

Comment:
Question) Is there a reason not to use the most recent version?
https://www.ecma-international.org/publications-and-standards/standards/ecma-262/

(I'm not too familiar with the ecosystem so maybe this is too bleeding edge though.. 😅 )

Reply:
I just copied the file from Armeria without deep thoughts. 😆 But I wanted to sync the configuration for the two projects.
https://github.com/line/armeria/blob/master/settings/eslint/eslintrc.js
We may create a separate repo to share it later.

Reply:
Sounds good to me 👍